FlowControlHomework/FlowControlHomework/FlowControl.cs

```diff
@@ -307,47 +307,52 @@ public void Switch()
             {
                 case "a":
                     Assert.IsTrue(true);
+                    break;
                 case "b":
                     Assert.IsTrue(false);
+                    break;
                 case "c":
                     Assert.IsTrue(false);
+                    break;
                 default:
                     Assert.IsTrue(false);
+                    break;
             }
 
             switch (usersChoice) // Fix me so I compile
-            { 
-                "a":
+            {
+                case"a":
                     Assert.IsTrue(true);
                     break;
-                "b":
+                case"b":
                     Assert.IsTrue(false);
                     break;
-                "c":
+                case"c":
                     Assert.IsTrue(true);
                     break;
                 default:
                     Assert.IsTrue(false);
-                    break;
-            }
-
+                break;
         }
 
+    }
+
 
         [TestMethod]
         public void Loops()
         {
 
-            int max = 12; // set me correctly so i reach the assert pass.
+            int max = 15; // set me correctly so i reach the assert pass.
             for (int i = 0; i <= max; i++)
             {
                 Console.WriteLine(true);
-                // DONT BE ALARMED, you dont need brackets if its only a single statement you want to execute. Im just exposing you to this because programmers do it.
-                // But don't do it yourself. It's not a good practice, especially for a beginner.
+                //DONT BE ALARMED, you dont need brackets if its only a single statement you want to execute. Im just exposing you to this because programmers do it.
+
+                //But don't do it yourself. It's not a good practice, especially for a beginner.
                 if (i == 15) Assert.IsTrue(true);
             }
 
-           int max2 = 17;
+            int max2 = 17;
```

Comment:
I made a mistake here. Where the comment says `if you reach here test will fail`, the assert should say `Assert.IsTrue(false)`.
Line 363 (original) or line 368 (your version).